tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py

```diff
@@ -325,7 +325,7 @@ def _wait_for_menu_item(self, menu_item_locator):
     def _wait_for_parent_section_not_displayed(self, menu_item):
         section = menu_item.find_element(By.XPATH, 'ancestor::section')
         Wait(self.marionette).until(
-            lambda m: section.location['x'] + section.size['width'] == 0)
+            lambda m: abs(section.location['x']) == abs(section.size['width']))
```

Comment:
IIRC, this wait is to check that the child panel moved to the right, isn't it? I believe it'd be safer to use >=, in case where the x position is 1 pixel away from the border. What do you think?